apps/example/options.tsx

```diff
@@ -17,6 +17,25 @@ export const options: NextAdminOptions = {
         display: ["id", "name", "email", "posts", "role", "birthDate"],
         search: ["name", "email", "role"],
         copy: ["email"],
+        filters: [
+          {
+            name: "is Admin",
+            active: false,
+            value: {
+              role: {
+                equals: "ADMIN",
+              },
+            },
+          },
+          {
+            name: "+1 posts",
+            value: {
+              posts: {
+                some: {},
+              },
+            },
```

Comment:
Maybe find something else since its not possible to filter by relationship count